cirq/contrib/qasm_import/_parser.py

```diff
@@ -11,11 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from typing import Dict, Optional
+from typing import Dict, Optional, List, Any, Iterable
 
 from ply import yacc
 
-from cirq import Circuit
+import cirq
+from cirq import Circuit, NamedQubit, CNOT
```

Comment:
`from cirq import CX` also works and it's more consitent with QASM using `cx`.

Reply:
sounds good!